backend/app/api/v1/lantern_api.py

```
@@ -61,11 +62,11 @@ async def create_lanterns(
         500: error_500
     }
 )
-async def get_lanterns(
+async def get_lantern_list(
         current_lantern_id: Optional[str] = Query(
             None,
-            description="현재 관람 중인 사용자의 랜턴 ID",
-            regex=r"^[가-힣a-zA-Z0-9]+-[0-9]+$"
+            description="현재 체험 중인 사용자 랜턴 ID",
+            regex = r"^[가-힣a-zA-Z0-9]+-[0-9]{4}$"
```

Flake8 (7.2.0)

[error] unexpected spaces around keyword / parameter equals

(E251)